CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Changelog for Critical Maps iOS
 
 # [Unreleased]
 
+### Added
+
+- Showing next ride event notification and annotation view on the map.
+
```

Comment:
Not sure if it should be in the changeling since it's not active yet. Maybe we net a column for internal changes that won't be part of the App Store changelogs

Reply:
But a next release could already have that feature, right? Or at least it would make sense IMO

Reply:
What are the things that are missing for adding it?

Reply:
I think we should have the Attribution banner in Settings before shipping and we also need to enable the Feature flag. The refactorings are not a ship blocker but we should create issues for them so that we don't forget about them

Reply:
Ok. Thats fine with me. So basically just the banner

Reply:
I updated the outdated check to use a timer and also moved some code around

Reply:
But please lets add the banner in a separate PR. Makes it easier to review

Reply:
Hahaha. No worries 😉